controllers/sync/policy_status_sync.go

```diff
@@ -131,17 +138,15 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 	// plc matches hub plc, then get events
 	eventList := &corev1.EventList{}
 	err = r.ManagedClient.List(ctx, eventList, client.InNamespace(instance.GetNamespace()))
+
 	if err != nil {
 		// there is an error to list events, requeue
 		return reconcile.Result{}, err
 	}
 	// filter events to current policy instance and build map
 	eventForPolicyMap := make(map[string]*[]policiesv1.ComplianceHistory)
-	rgx, err := regexp.Compile(`(?i)^policy:\s*([A-Za-z0-9.-]+)\s*\/([A-Za-z0-9.-]+)`)
-	if err != nil {
-		// regexp is wrong, how?
-		return reconcile.Result{}, err
-	}
+	// panic if regexp invalid
+	rgx := regexp.MustCompile(`(?i)^policy:\s*([A-Za-z0-9.-]+)\s*\/([A-Za-z0-9.-]+)`)
```

Comment:
I like this, if we mess up the regexp then it should panic during a test, which will be more obvious than it was before 👍